e2e/components/tabs/tabs.e2e.ts

```diff
@@ -0,0 +1,24 @@
+describe('tabs', function () {
+  describe('basic behavior', function () {
+    beforeEach(function() {
+      browser.get('/tabs');
+    });
+    it('should change tabs when the label is clicked', function () {
+      element(by.css('.md-tab-label:nth-of-type(2)')).click();
```

Comment:
Rather than using `nth-of-type`, you can do `element.all().get(index)`, e.g.
```ts
let tabLabels = element.all(by.css('.md-tab-label'));
tabLabels.get(2).click();
...
```